Frameworks/include/IDisplayTextureImpl.h

```diff
@@ -0,0 +1,57 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+#pragma once
+
+#import <Starboard.h>
+#import <CoreGraphics/CGImage.h>
+#import "CGContextInternal.h"
+#import "CACompositor.h"
+
+#import <memory>
+
+//TODO #1337 - Remove this when Compositor refactoring is complete and ready.
+class IDisplayTextureImpl : public IDisplayTexture {
```

Comment:
This is something that's going away, the reason the naming is so weird is that, first it stands out.
Second there is already DisplayTexture defined.

---
In reply to: [87102669](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r87102669) [](ancestors = 87102669,87102644)